ArcGISRuntimeSDKQt_QMLSamples/Layers/TileCacheLayer/README.md

```diff
@@ -0,0 +1,38 @@
+# Tile cache layer
+Load an offline copy of a tiled map service as a basemap.
+
+![](screenshot.png)
+
+## Use case
+Constructing an 'ArcGISTiledLayer' from a local tile package (.tpk & .tpkx) allows you to use that basemap tiled service when the application is offline. Dividing a raster into tiles allows the map to provide relevant tiles and level of detail to the user when panning and zooming. For example, when working in an environment that has no connectivity, this could provide access to a map for navigating your surroundings.
```

Comment:
QML doesn't really have constructors so we generally change the wording from the common design. In this case, something like "Creating an ArcGISTiledLayer..." would be more appropriate.